### PR TITLE
Ankit | Test | PROD - E-Invoice column is showing in the get all page even when setting is off for the E-Invoice

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwxrtrd
https://app.clickup.com/t/86cwx1wha
https://app.clickup.com/t/86cwx1v40

### CODE REVIEW

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 1]
```diff
@@ -526,6 +526,10 @@ button.btn.btn-filter {
 }
 
 .all-table {
+    table {
+        z-index: -1;
+        position: relative;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/contact/contact.component.scss]: use class

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 2]
```diff
@@ -375,7 +374,7 @@ export class VoucherListComponent implements OnInit, OnDestroy {
 
             }
         });
-
+        this.isCompany = this.generalService.currentOrganizationType === OrganizationType.Company;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts]: check

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/contact.component.scss (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/contact/contact.component.scss]: not available class for z-index: -1;
position DONE

File: apps/web-giddh/src/app/vouchers/list/list.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts]: Done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**